.github/workflows/ci-conda-workflow.yml

```
@@ -169,6 +162,14 @@ jobs:
         if [ -n "${XSPECVER}" ]; then XSPECTEST="-x -d"; fi
```

```suggestion
        if [ "$RUNNER_OS" != "macOS" ]; then DS9TEST=" -d"; fi
        if [ -n "${XSPECVER}" ]; then XSPECTEST="-x${DS9TEST}"; fi
```
I don't know if this makes things easier or harder to read, so feel free to ignore this suggestion.

---

I would hope that this is all a temporary change so I don't feel like over-engineering the solution..........

---

Agreed. And I hope so too!